tests/rptest/tests/group_membership_test.py

```diff
@@ -0,0 +1,46 @@
+# Copyright 2021 Vectorized, Inc.
+#
+# Use of this software is governed by the Business Source License
+# included in the file licenses/BSL.md
+#
+# As of the Change Date specified in that file, in accordance with
+# the Business Source License, use of this software will be governed
+# by the Apache License, Version 2.0
+
+import time
+
+from ducktape.mark import matrix
+from ducktape.mark.resource import cluster
+from ducktape.utils.util import wait_until
+
+from rptest.clients.types import TopicSpec
+from rptest.tests.redpanda_test import RedpandaTest
+from rptest.clients.kcl import KCL
+
+
+class ListGroupsReplicationFactorTest(RedpandaTest):
+    """
+    We encountered an issue where listing groups would return a
+    coordinator-loading error when the underlying group membership topic had a
+    replication factor of 3 (we had not noticed this until we noticed that
+    replication factor were defaulted to 1). it isn't clear if this is specific
+    to `kcl` but that is the client that we encountered the issue with.
```

Comment:
kcl just spits an empty `ListGroups` request to all brokers, fwiw.

Reply:
yep, that's how I understood it.